src/java.desktop/share/classes/javax/swing/AbstractButton.java

```diff
@@ -2146,9 +2146,9 @@ protected void init(String text, Icon icon) {
 
 
     /**
-     * This is overridden to return false if the current <code>Icon</code>'s
-     * <code>Image</code> is not equal to the
-     * passed in <code>Image</code> <code>img</code>.
+     * Returns true if one of this AbstractButton's icon or disabled icon is derived from
+     * ImageIcon and it's Image is equal to the image passed in; false
+     * otherwise.
```

Comment:
Guess it should be "Returns true if the button icon for present button state is an instance of ImageIcon and the image it contains is same as the <code>image</code>"

Reply:
same for JLabel..

Reply:
Does it should be <code>img</code> in stead of <code>image</code> ?

Reply:
image

Reply:
ok